### PR TITLE
UD-1022 add new user API

### PR DESCRIPTION
**Pull request by olexii4**
@ashumilova @benoitf  WDYT?

### CODE REVIEW

**Comment by olexii4** [comment]
**FYI**: I guess we should move **che-user.factory** with own requests to **dashboard** and leave in **che-dashboard**  only  dummy  user-factory.  But this should be in several PRs.

File: src/components/attribute/scroll/che-list-on-scroll-bottom.directive.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export class CheListOnScrollBottom {
   link(scope, element, attrs) {
     var raw = element[0];
     element.bind('scroll', function () {
-      if (raw.scrollTop + raw.offsetHeight === raw.scrollHeight) {
```
Comment (benoitf) [inline src/components/attribute/scroll/che-list-on-scroll-bottom.directive.js]: Could you please move this fix to a separate commit as it's not part of users

**Comment by benoitf** [comment]
@olexii4 yes getUsers method shouldn't belong to cheAPI but be in codenvyAPI/codenvyUser but as long as you move it in a reasonable timeframe it's ok for me

**Comment by ashumilova** [comment]
+1 for moving to codenvyAPI

**Comment by ashumilova** [comment]
+1
